src/store/store.js

```diff
@@ -1,15 +1,14 @@
 import { createStore, combineReducers, applyMiddleware } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
 
-import loginReducer from './login/reducer';
-import registerReducer from '.register/reducer';
-import thunkMiddleware from './redux-thunk';
+import loginReducer from './login/reducers';
+
+import thunkMiddleware from 'redux-thunk';
```

Comment:
I suggest moving this to be part of your first group of imports since it is not a local import